docs/install.rst

```diff
@@ -64,13 +70,26 @@ download and run it as follows::
   user@webserver:~$ chmod a+x ./certbot-auto
   user@webserver:~$ ./certbot-auto --help
 
-.. hint:: The certbot-auto download is protected by HTTPS, which is pretty good, but if you'd like to
-          double check the integrity of the ``certbot-auto`` script, you can use these steps for verification before running it::
+To check the integrity of the ``certbot-auto`` script,  
+you can use these steps::
+
+	  
+	    user@webserver:~$ wget -N https://dl.eff.org/certbot-auto.asc
+	    user@webserver:~$ gpg2 --keyserver pool.sks-keyservers.net --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
+	    user@webserver:~$ gpg2 --trusted-key 4D17C995CD9775F2 --verify certbot-auto.asc certbot-auto
+
+	    gpg: Signature made Wed 02 May 2018 05:29:12 AM IST
```

Comment:
I worry that an unfamiliar user might think the output should match exactly.

What do you think about adding a line above this saying something like:

"The output of the last command should look something like::"

Reply:
Its 0KAY

Ya i think 
it make sense, few might think like that
will add that line